lib/resources/r4/condition.yaml

```diff
@@ -117,3 +184,27 @@ fields:
       - display: ICD-10-CM
         system: http://hl7.org/fhir/sid/icd-10-cm
         info_link: https://hl7.org/fhir/icd.html
+
+- name: onsetDateTime
+  required: 'No'
+  cardinality: 0..1
+  type: dateTime
+  description: Estimated or actual date the condition began, in the opinion of the clinician.
+  example: |
+    {
+      "onsetDateTime": "2000-03-10T02:00:00.00+00:00"
+    }
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.onset_x_
+
+- name: abatementDateTime
+  required: 'No'
+  cardinality: 0..1
+  type: dateTime
+  description: Indicates the date and time the condition was resolved or has gone into remission.
+  example: |
+    {
+      "abatementDateTime": "2010-11-08T02:00:00.00+00:00"
+    }
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.abatement_x_
+
+- name: note
```

Comment:
Do we miss documentation here?

Reply:
My bad. Added [here](https://github.com/nishanthnamana/fhir.cerner.com/commit/91fee55a512ed5f620c71861b3940e8479c3f61a#diff-abb99d2139d79f8a2a8f1d214f5f6882R210).